docs/proposals/21-ipv6-singlestack-local.md

```diff
@@ -261,7 +261,7 @@ Hence, gardenlet explicitly enables the corresponding kernel setting for shoot w
 
 The `docker.io` registry doesn't support pulling images over IPv6 (see [Beta IPv6 Support on Docker Hub Registry](https://www.docker.com/blog/beta-ipv6-support-on-docker-hub-registry/)).
 
-Container images from `docker.io` used on shoots and seeds are rewritten to `registry.ipv6.docker.com` if the corresponding `ipFamilies` field specifies IPv6 singe-stack.
+Container images from `docker.io` used on shoots, seeds and extensions are hard-coded to `mirror.gcr.io` in all respective `images.yaml`. The Google container registry supports dual-stack access and has no rate-limit.
```

Comment:
Isn't exactly this missing from this PR? I still see images from Docker hub used in https://github.com/gardener/gardener/blob/master/charts/images.yaml :)

Reply:
yes this is true, since these images are not yet deployed when performing a `make kind-up`, they were not yet addressed.
The next PR, which handles `make gardener-up` will address this an change all images that we currently use from docker.io